shared/common-adapters/header-hoc.native.js

```diff
@@ -13,6 +13,7 @@ export const HeaderHocHeader = ({
   hideBackLabel,
   title,
   onCancel,
+  customCancelText,
```

Comment:
Since some popups purposely use the text "Close" instead of "Cancel", I added a prop in `HeaderHoc` for this case.